data_steward/cdr_cleaner/cleaning_rules/clean_mapping.py

```diff
@@ -69,7 +69,8 @@ def __init__(self, project_id, dataset_id, sandbox_dataset_id):
                          affected_datasets=[cdr_consts.RDR],
                          project_id=project_id,
                          dataset_id=dataset_id,
-                         sandbox_dataset_id=sandbox_dataset_id)
+                         sandbox_dataset_id=sandbox_dataset_id,
+                         affected_tables=['_mapping'])
```

Comment:
@lrwb-aou Clean_mapping uses dynamic logic to get the affected tables, but it has to get two different sets of affected tables. can you suggest some ideas to handle this issue?

Reply:
@ksdkalluri have you considered setting this to a function that returns a list of table names based on the tables in `resource_files/fields` directory?  The validation function for this class may need to implement its own method instead of using the base class method.  This potentially affects all `_mapping` and `_ext` tables.  So I think a list of those should suffice.  You can use a helper method to generate that list.  Does this make sense?  Other suggestions or considerations?

Reply:
yes, I did consider getting tables from fields but the get_tables method has logic that finds the tables in the dataset (mapping or ext) and finds the associated CDM tables. which uses bq to run a query to get all the tables in a dataset. I am starting to think we might also pass the client as the param for the get_affected_tables.